spec/models/player_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'models/player'
+
+module RubyConnect
+  describe Player do
+    let(:player) { Player.new color: :red, name: "John", game: 1 }
+
+    context '#make_move' do
```

Comment:
I believe the convention is to use `describe` instead of `context` in this case.